dojopuzzles/problems/views.py

```diff
@@ -20,3 +20,8 @@ def problem_select(request, problem_id):
     problem.select()
     request.session["problem_selected"] = problem.id
     return redirect(problem)
+
+
+def problem_list(request):
+    problems = Problem.objects.publisheds()
+    return render(request, 'problems/list.html', {'object_list': problems})
```

Comment:
**suggestion (formatting)**: I noticed that you used single-quotes here. There is no standard for code formatting in the project (yet), but I suggest that you run [`black`](https://github.com/psf/black) on your Python files, so code have a consistent format.

Reply:
Thanks for the hint, i change that.